packages/slice-machine/src/hooks/useEditorContent.ts

```diff
@@ -15,12 +16,12 @@ function useEditorContentOnce({
     slice.mock?.find((m) => m.variation === variationID) ||
     defaultSharedSliceContent(variationID);
 
-  const apiContent = {
+  const apiContent = useMemo(() => ({
     ...(renderSliceMock(Slices.fromSM(slice.model), editorContent) as object),
     id: slice.model.id,
-  };
+  }), []);
```

Comment:
Does adding the dependencies here trigger the bug?

Reply:
you're right! Updated